pkg/result/processors/skip_dirs.go

```diff
@@ -26,7 +35,7 @@ type SkipDirs struct {
 	pathPrefix       string
 }
 
-func NewSkipDirs(patterns []string, log logutils.Log, args []string, pathPrefix string) (*SkipDirs, error) {
+func NewSkipDirs(log logutils.Log, patterns, args []string, pathPrefix string) (*SkipDirs, error) {
```

Comment:
I believe the previous argument arrangement was better:
```suggestion
func NewSkipDirs(patterns []string, log logutils.Log, args []string, pathPrefix string) (*SkipDirs, error) {
```

This way, it's harder to confuse `patterns` with `args` when calling `NewSkipDirs`.

Compare:

```go
skipDirsProcessor, err := processors.NewSkipDirs(skipDirs, log.Child(logutils.DebugKeySkipDirs), args, cfg.Output.PathPrefix)
```

vs

```go
skipDirsProcessor, err := processors.NewSkipDirs(log.Child(logutils.DebugKeySkipDirs), args, skipDirs, cfg.Output.PathPrefix)
```

Reply:
I think we need to have a homogenous way to handle the logger, the logger is like the context and should be in the first place.
The argument names are enough not to be confusing.